src/js/controller/LayersListController.js

```diff
@@ -130,6 +130,14 @@
       layerNameEl.setAttribute('title', layer.getName());
       layerNameEl.setAttribute('rel', 'tooltip');
     }
+    var opacity = layer.getOpacity();
+    if (opacity == 1) {
+      layerItem.getElementsByClassName('layer-item-opacity')[0].style.color = '#ffd700';
```

Comment:
In the codebase we usually use querySelector to find elements, could we replace this with:

```layerItem.querySelector('.layer-item-opacity')```